src/styles.css

```diff
@@ -3,4 +3,6 @@
 
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```

Comment:
We should not be changing fonts, TailwindCSS provides us with one we've been using.